src/app.js

```diff
@@ -0,0 +1,64 @@
+const express = require("express");
+const app = express();
+const swaggerJsDoc = require("swagger-jsdoc");
+const swaggerUi = require("swagger-ui-express");
+
+const port = process.env.PORT || 5000;
+
+// Extended: https://swagger.io/specification/#infoObject
+const swaggerOptions = {
+    swaggerDefinition: {
+        info: {
+            title: "Customer API",
+            description: "Customer API Information",
+            contact: {
+                name: "Amazing Developer"
+            },
+            servers: ["http://localhost:5000"]
+        }
+    },
+    // ['.routes/*.js']
+    apis: ["src/app.js"]
+};
+
+const swaggerDocs = swaggerJsDoc(swaggerOptions);
+app.use("/api-docs", swaggerUi.serve, swaggerUi.setup(swaggerDocs));
+
+// Routes
+/**
+ * @swagger
+ * /customers:
+ *  get:
+ *    description: Use to request all customers
+ *    responses:
+ *      '200':
+ *        description: A successful response
+ */
+app.get("/customers", (req, res) => {
```

Comment:
What is this for? I notice that this endpoint isn't documented on the auto-generated API, so I assume it's boilerplate stuff that's auto-generated?

I think the reason why this is not included in the auto-documentation is because it isn't included in Swagger options in `config.js`.

Reply:
This was a mistake.